include/deal.II/matrix_free/portable_matrix_free.h

```diff
@@ -635,11 +650,11 @@ namespace Portable
       MemorySpace::Default::kokkos_space::execution_space>::member_type;
 
     using SharedView1D = Kokkos::View<
-      Number *,
+      Number **,
       MemorySpace::Default::kokkos_space::execution_space::scratch_memory_space,
       Kokkos::MemoryTraits<Kokkos::Unmanaged>>;
     using SharedView2D = Kokkos::View<
```

Comment:
```suggestion
    using SharedViewGradients = Kokkos::View<
```

Reply:
We need to be a little careful about not running out of shared memory for large polynomial degrees and a large number of components. Of course, this would already be a problem and we should tackle it separately.